spec/controller/post_spec.rb

```diff
@@ -12,10 +12,10 @@
     expect(response).to render_template('index')
   end
 
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you rename the containing folder for these tests from `controller` to `requests` as is the convention since we are testing requests.

  Also, a `spec/requests` folder is important for assisting certain configurations